### PR TITLE
feat: update dependencies

### PR DESCRIPTION
**Pull request by ohassine**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Updating some dependencies to latest version  

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by MohamadJaara — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4314](https://10.10.124.134/job/AR-build-pipeline/4314/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(9448344)(9448344)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(9448344)(9448344)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [4325](https://10.10.124.134/job/AR-build-pipeline/4325/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(9448938)(9448938)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(9448938)(9448938)-staging-compat.apk)

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -30,22 +30,22 @@ kover = "0.7.6"
...
 androidx-splashscreen = "1.0.1"
 androidx-workManager = "2.9.0"
-androidx-browser = "1.7.0"
+androidx-browser = "1.8.0"
 androidx-biometric = "1.1.0"
 androidx-startup = "1.1.1"
 
 # Compose
-composeBom = "2024.03.00" # TODO check if in new version [anchoredDraggable] is available
+composeBom = "2024.04.01" # TODO check if in new version [anchoredDraggable] is available
```
Comment (Garzas) [inline gradle/libs.versions.toml]: We can remove this comment, because `anchoredDraggable` is already implemented

**Review by Garzas — APPROVED** [review]
We can remove one comment :)

**Comment by AndroidBob** [comment]
Build [4328](https://10.10.124.134/job/AR-build-pipeline/4328/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.7.0(9454296)(9454296)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.7.0(9454296)(9454296)-staging-compat.apk)

**Pull request merged**